File: app/src/main/java/com/andromeda/ara/MainInteractionSessionService.java
package com.andromeda.ara;

import android.os.Build;
import android.os.Bundle;
import android.service.voice.VoiceInteractionSession;
import android.service.voice.VoiceInteractionSessionService;

import androidx.annotation.RequiresApi;

@RequiresApi(api = Build.VERSION_CODES.M)
public class MainInteractionSessionService extends VoiceInteractionSessionService {
    @Override
    public VoiceInteractionSession onNewSession(Bundle args) {
        return new VoiceUi(this);
    }
}

File: app/src/main/java/com/andromeda/ara/MainInteractionSessionService.kt
package com.andromeda.ara

import android.os.Build
import android.os.Bundle
import android.service.voice.VoiceInteractionSession
import android.service.voice.VoiceInteractionSessionService

import androidx.annotation.RequiresApi

@RequiresApi(api = Build.VERSION_CODES.M)
class MainInteractionSessionService : VoiceInteractionSessionService() {
    override fun onNewSession(args: Bundle): VoiceInteractionSession {
        return VoiceUi(this)
    }
}
